frontend/src/index.js

```diff
@@ -10,9 +10,15 @@ import { ApolloProvider } from "@apollo/react-hooks";
 import { InMemoryCache } from "apollo-cache-inmemory";
 import { HttpLink } from "apollo-link-http";
 
+// Environment detection
+let apiUrl = "https://api.foreignlanguagereader.com/api/";
+if (window.location.hostname === "localhost") {
+  apiUrl = "http://localhost:4000/api/";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Delete `;`](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5099894)

Reply:
This is the opposite of the ESlint rules.